pilot/pkg/keycertbundle/keycertbundle.go

```diff
@@ -0,0 +1,111 @@
+// Copyright Istio Authors
```

Comment:
There is already keycertbundle.go https://github.com/istio/istio/blob/master/security/pkg/pki/util/keycertbundle.go. I think it would be nice to integrate your changes here.

Reply:
I have been aware of that file, they are not same, the interface diffs much.

Reply:
ok

Reply:
Is there a plan to phase that code out? It appears that we have two files with similar names that could be confusing. Perhaps the new file could be called keyCertBundleWatcher or something?